docs/index.md

```diff
@@ -641,6 +643,52 @@ The middleware, if successful, will respond with a `204 No Content` HTTP status
 
 Both `jsonApiKoa` and `jsonApiExpress` expose a `/bulk` endpoint which can be used to execute multiple operations. The request must use the `PATCH` method, using the JSON payload [shown earlier](#running-multiple-operations).
 
+### Serverless Functions
+
+Since version `1.2.0`, Kurier can be used inside serverless functions.
+
+#### Using Vercel + Next.js
+
+If you want your Next.js API to work with Kurier in a Vercel-hosted environment, you'll need to create a _generic route_ in your `pages/api` directory, like this:
+
+```
+pages/
+|__ api/
+    |__ [...kurier].js
+```
+
+In case you don't want all of your endpoints to go through Kurier, you can namespace them:
+
+```
+pages/
+|__ api/
+    |__ kurier/
+        |__ [...kurier.js]
+```
+
+In the `[...kurier.js]` file, you can import the `jsonApiVercel` middleware to handle the resource endpoints automatically, just like `jsonApiKoa` and `jsonApiExpress` do:
+
+```js
+import { Application, jsonApiVercel } from 'kurier';
+import { knex } from 'knex';
+
+const app = new Application({
+  // The `namespace` property must match your directory structure under `pages/api`.
+  namespace: 'api/kurier',
+  // ...the rest of the Application properties (types, processors, etc.).
+});
+
+// You can also add a database connection with Knex.
+app.services.knex = knex({ /* Your DB configuration */ });
+
+// Export the middleware result so Next.js can handle Kurier endpoints.
+export default jsonApiVercel(app);
+```
+
+#### Using Vercel Functions
+
+For Vercel Functions, the procedure is the same as above. Instead of creating `/pages/api/[...kurier.js]`, create `/api/[...kurier.js]`.
```

Comment:
```suggestion
For Vercel Functions, the procedure is the same as above. Instead of creating `/pages/api/[...kurier].js`, create `/api/[...kurier].js`.
```